test/ui/11 - errors/problem-with-this-service.spec.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const config = require('../ui-test-config');
+
+describe.only('Error - 500', () => {
+
+  describe('FR-ERR-29 (FBISCC-76) - \'Service failure\' error page', () => {
+
+    beforeEach(async() => {
+      await page.goto(baseURL + '/question');
```

Comment:
Can you test with an email failure on the feedback route, instead of the main route? This way we can save the main route for the special 'form not submitted' error page that the designers are working on. You can still use `config.notifyFailureEmail` to force a failure in the feedback route..